### PR TITLE
Changes of ReportSystem ver.2

### PR DESCRIPTION
**Pull request by Allof02**
Modified ReportScreen so now itself is a jpanel
Add Icon to ReportScreen Jframe
RENAME ReportInteractor to ReportInteract
FIX ReportInteract, ReportController
ADD getMethods to FileReportHistroy
ADD ReportTest.java to Test folder

### CODE REVIEW

**Review by prajogt — DISMISSED** [review]

File: src/main/java/report_screens/FileReportHistory.java [hunk 1]
```diff
@@ -106,5 +106,13 @@ public boolean existsReportByReporterAndReview(String reporter_username, String
...
         return Objects.equals(corresponding_reporter, reporter_username);
     }
+
+    public Map<String, ReportDsRequestModel> getSave_reports() {
+        return save_reports;
+    }
+
+    public Map<String, String> getCheck_reports() {
+        return check_reports;
+    }
```
Comment (prajogt) [inline src/main/java/report_screens/FileReportHistory.java:116]: Consider renaming check_reports to reviewOwner or something like that so the variable says what it contains rather what it is used for.

Comment (Allof02) [reply src/main/java/report_screens/FileReportHistory.java:116]: sure

**Review by henryklinck — DISMISSED** [review]

**Comment by Allof02** [comment]
Fix ReportScreen, LabelTextPanel
Moved ProfileScreen to profile_screens package

**Review by JarodGratton — APPROVED** [review]

File: src/main/java/report_screens/ReportScreen.java [hunk 2]
```diff
@@ -13,9 +13,9 @@
 
 
 
-public class ReportScreen implements ActionListener {
+public class ReportScreen extends JPanel implements ActionListener {
```
Comment (foreverwoods) [inline src/main/java/report_screens/ReportScreen.java:16]: Good change to make it a JPanel and switch to JTextArea.

**Review by foreverwoods — APPROVED** [review]

**Review by davatana — APPROVED** [review]

**Pull request merged**